db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20161104013922) do
+ActiveRecord::Schema.define(version: 20161110011213) do
```

Comment:
Sooo, if github really sees a conflict here, even though my cli git client doesn't, but gh says "resolve it on the command line", do I just do the merge on cli, then submit a PR for master or something?